lib/Doctrine/DBAL/Tools/Console/Command/PingCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Doctrine\DBAL\Tools\Console\Command;
+
+use Doctrine\DBAL\Connection;
+use LogicException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Throwable;
+use function is_numeric;
+use function sleep;
+use function sprintf;
+
+class PingCommand extends Command
```

Comment:
Should we mark this as `final`?

Reply:
Let's mark that one as unresolved until we get feedback from other maintainers on the decoration vs extension debate.

Reply:
We should, makes things clearer IMHO

Reply:
I couldn't think of a way to avoid extension here, let's resolve this.